docs/fxa-engineering/fxa-sub-platform.md

```diff
@@ -10,28 +10,35 @@ sidebar_label: Subscription Platform
 
 To begin working on the subscription platform in the FxA codebase, you will need access to a Stripe account for private and public API developer keys.
 
-If you're a Mozilla employee, you can request a private key from the FxA Subscription Platform team. Otherwise, you can create your own Stripe account to use for testing that is not linked to any bank account information. These keys should be taken from Stripe's test environment.
+If you're a Mozilla employee, you can request access to the Stripe dev (and/or stage) account, created for the FxA Subscription Platform team to easily connect with fake products and plans. Otherwise, you can create your own Stripe account to use for testing that is not linked to any bank account information with your own products and plans. These keys should be taken from Stripe's test environment which you can verify by checking that the key includes the word `test`.
 
-The `fxa-payments-server` needs the Stripe public key and communicates with the `fxa-auth-server` that requires a Stripe private key. Configuration details can be found below.
+The `fxa-payments-server` needs the Stripe public key (`pk`) and communicates with the `fxa-auth-server` that requires a Stripe private or secret key (`sk`). These can be found in the Stripe Dashboard, and configuration details can be found below.
```

Comment:
We have in the past given out restricted keys (`rk_`), we might want to reflect that since we may do it again or use those even in our dev env at some point.

Reply:
@bbangert I've added a note about that along with a `Current as of` note with a date at the top, similar to other pages I've seen in this hub.